lib/reverse_sentence.rb

```diff
@@ -1,6 +1,37 @@
 # A method to reverse the words in a sentence, in place.
+
 # Time complexity: ?
 # Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+
+# min = my_sentence[0]
+# max = my_sentence[-1]
+# my_arr = #["yoda"__"is"___"awesome!"].dup and add in spaces to new array[?]
+# my_arr = ["yoda", "__", "is", "___", "awesome"]
+# my_new_arr = []
+# loop thru 
+# first = my_arr[0] 
+# last = my_arr[-1]
+# first, last = last, first 
+# last += 1
+# first -= -1
+# shovel into my_new_array 
+# in a perfect world, would return in order we want
+
+def reverse(sentence)
```

Comment:
This method reverses a string, it doesn't reverse the words in a sentence.  You're on the right track.

Consider:

1. Reverse the string
2. Loop through reversing each word in the sentence.